package/rules/swc.js

```diff
@@ -0,0 +1,73 @@
+const { resolve } = require('path')
+const { realpathSync } = require('fs')
+
+const {
+  source_path: sourcePath,
+  additional_paths: additionalPaths
+} = require('../config')
+const { isProduction } = require('../env')
+
+module.exports = [
+  {
+    test: /\.(js|jsx|mjs|coffee)?(\.erb)?$/,
```

Comment:
Need to split into ts and js parsers. Can technically run js through typescript parser but I've had some TS specific errors (non-type checking) when trying that against JS files